src/Tasks.UnitTests/CSharpParserUtilitites_Tests.cs

```diff
@@ -168,136 +193,202 @@ public void ScopelessNamespaceSyntaxError()
             AssertParse("namespace i; namespace j { class a {} }", null);
         }
 
-        [Fact]
-        public void AssemblyAttributeBool()
+        [Theory]
+        [InlineData("[assembly :AssemblyDelaySign(false)] namespace i { class a { } }", "i.a")]
+        [InlineData("[assembly :AssemblyDelaySign(false)] namespace i; class a { }", "i.a")]
+        public void AssemblyAttributeBool(string fileContents, string expected)
         {
-            AssertParse("[assembly :AssemblyDelaySign(false)] namespace i { class a { } }", "i.a");
+            AssertParse(fileContents, expected);
         }
 
-        [Fact]
-        public void AssemblyAttributeString()
+        [Theory]
+        [InlineData("[assembly :MyString(\"namespace\")] namespace i { class a { } }", "i.a")]
+        [InlineData("[assembly :MyString(\"namespace\")] namespace i; class a { }", "i.a")]
+        public void AssemblyAttributeString(string fileContents, string expected)
         {
-            AssertParse("[assembly :MyString(\"namespace\")] namespace i { class a { } }", "i.a");
+            AssertParse(fileContents, expected);
         }
 
-        [Fact]
-        public void AssemblyAttributeInt()
+        [Theory]
+        [InlineData("[assembly :MyInt(55)] namespace i { class a { } }", "i.a")]
+        [InlineData("[assembly :MyInt(55)] namespace i; class a { }", "i.a")]
+        public void AssemblyAttributeInt(string fileContents, string expected)
         {
-            AssertParse("[assembly :MyInt(55)] namespace i { class a { } }", "i.a");
+            AssertParse(fileContents, expected);
         }
 
-        [Fact]
-        public void AssemblyAttributeReal()
+        [Theory]
+        [InlineData("[assembly :MyReal(5.5)] namespace i { class a { } }", "i.a")]
+        [InlineData("[assembly :MyReal(5.5)] namespace i; class a { }", "i.a")]
```

Comment:
Additional cases here aren't _bad_ but I'm not sure they're providing a ton of value. Would any of these have broken at any point while you were developing the fix?

Reply:
They wouldn't have, but it didn't feel right leaving perfectly valid tests unmodified WRT file-scoped namespaces